docs : Adding warning message for migration of ApplicationSet into Argo CD (#548)
argoproj/applicationset: README.md

# Argo CD ApplicationSet Controller 

> ### :warning: This project code has been moved to the main Argo CD repository
> 

Reviewer: ```suggestion
## :warning: This project code has been moved to the main Argo CD repository

```

> This repository will no longer be active.It is now merged with ArgoCD and will be released along with it. Further development will happen in [Argo CD](https://github.com/argoproj/argo-cd).

Reviewer: ``
```suggestion
This repository is no longer active. ApplicationSet has been merged with Argo CD and will be released along with it. Further development will happen in [Argo CD](https://github.com/argoproj/argo-cd).
```
